DQM/GEM/plugins/GEMEfficiencyAnalyzer.cc

```diff
@@ -558,21 +567,29 @@ GEMEfficiencyAnalyzer::StartingState GEMEfficiencyAnalyzer::buildStartingState(
     }
   }
 
-  found &= state.isValid();
-
-  if (MuonHitHelper::isGEM(det_id)) {
-    const GEMDetId start_id{det_id};
+  if (found) {
```

Comment:
This check is useless...

Reply:
I worried about the case `TransientTrack::outermostMeasurementState`  or (`innermostMeasurementState`) returns invalid state. Is this an unnecessary worry...?

Reply:
I mean, replace the three lines with just
```C++
  found &= state.isValid();
```
`if found` here is the same as `found &=`